front/src/pods/student/student.container.tsx

```diff
@@ -27,22 +29,31 @@ export const PlayerContainer = () => {
     setSocket(localSocket);
 
     localSocket.on(SocketOuputMessageLiteral.MESSAGE, msg => {
-      console.log(msg);
-
       if (msg.type) {
         const { type, payload } = msg;
 
         switch (type) {
           case SocketReceiveMessageTypes.APPEND_TEXT:
             appendToLog(payload);
             break;
+          case SocketReceiveMessageTypes.STUDENT_GET_FULL_CONTENT:
+            appendToLog(payload);
```

Comment:
If we get the full content we should replace, not append

Reply:
No replace or set function on use-log.hook, so I was unsure if it was deliberate or it wasn't needed until now. New setLog functionality created.